amplify/backend/api/YourClearWayAPI/schema.graphql

```diff
@@ -98,20 +108,44 @@ type Question
         { allow: private, provider: iam, operations: [read] } # allow all authenticated users (Cognito User Pools) and IAM roles to read questions
     ]) {
     id: ID!
-    text: String!
-    tags: [Tag]
+    title: String! # short title of the question
+    questionText: String! # the content of the question
+    rewardText: String! # statement if the question is completed
+    difficulty: Difficulty!
+    cost: Cost!
+    type: Type!
+    tags: [Tag!]!
```

Comment:
just to clarify, the mobile app measure list has "criteria for qualification" and "additional information" sections, are those `title` and `questionText` respectively?

Reply:
So `title` is not on the list yet, but the designer will populate these fields on the list. `title` shows the gist of the question on the frontend, `questionText` is the content of the question, and `rewardText` is the phrase to show after completing the question.

Reply:
"additional information" will not be used in the app at least for now. `questionText` will be the "criteria for qualification", and`title` will be provided by the designers to show up on the frontend.